tests/test_ceo_dispatcher.py

```
@@ -419,16 +420,18 @@ def test_execution_request_has_worktree_path():
 
 def test_execution_request_worktree_path_excluded_from_dump():
     """worktree_path is internal — must not leak in model_dump()."""
+    test_worktree_path = "/tmp/wt-test"  # noqa: S108 - test-only placeholder path
+    test_github_token = "test-token-value"  # noqa: S106 - test-only placeholder, not a real credential
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "test_github_token"

(S105)